lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -144,6 +168,15 @@ def start
       end
     end
 
+    def start
+      super
+
+      init_api_client()
```

Comment:
This call sets up authorization. Isn't that needed for "fetch_metadata" to work?